grails-app/domain/org/pih/warehouse/invoice/Invoice.groovy

```diff
@@ -145,6 +146,10 @@ class Invoice implements Serializable {
         return InvoiceStatus.PENDING
     }
 
+    Float getTotalPrepaidValue() {
```

Comment:
Could you explain this? I was expecting something like the following. In my mind if there's a prepayment invoice, that includes all items and adjustments from the order at the prepayment rate. You can't have a prepayment invoice item that is not part of a prepayment invoice.
```
BigDecimal getTotalPrepaidValue() {
   return (invoiceType?.code == InvoiceTypeCode.PREPAYMENT_INVOICE) ? invoiceItems.sum { it.totalAmount } : 0
}
```

Reply:
That's right. But in this case we want to get the value of prepayment for the final invoice.